Examples/UtiliKit-iOSExample/Container/ContainerExampleViewController.swift

```diff
@@ -21,26 +23,47 @@ class BaseContainerViewController: UIViewController {
     private lazy var controllerA = ViewControllerA()
     private lazy var controllerB = ViewControllerB()
     
+    private var interactionController: HorizontalPanGestureInteractiveTransition?
+    private var animationController = WipeTransitionAnimator(direction: .leftToRight)
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
         
         containerViewController.managedChildren = [Child(identifier: "A", viewController: controllerA),
                                                    Child(identifier: "B", viewController: controllerB)]
 
+        //Configure the the container view controller
         containerViewController.willMove(toParent: self)
         addChild(containerViewController)
         containerView.addSubview(containerViewController.view)
         containerViewController.view.frame = containerView.bounds
         containerViewController.didMove(toParent: self)
+        
+        //Configure the interaction controller for the example
+        interactionController = HorizontalPanGestureInteractiveTransition(in: view) { [weak self] recognizer in
+            guard let self = self, let currentController = self.containerViewController.visibleController else { return }
+            
+            if recognizer.velocity(in: recognizer.view).x > 0 {
+                guard let previousChild = self.containerViewController.child(preceding: currentController) else { return }
+                self.animationController.transitionDirection = .leftToRight
+                self.containerViewController.transitionToController(for: previousChild)
+            } else {
```

Comment:
I don't know why I can't reply to the above comment - but I left it as user defined so you could make a decision. In theory you could specify RTL in one of those languages, but it may not make sense depending on which screen you are transitioning to/from.

As to this point, yea I'll split them up.